src/OrbitGl/App.cpp

```diff
@@ -691,36 +691,39 @@ ErrorMessageOr<void> OrbitApp::SavePreset(const std::string& filename) {
     filename_with_ext += ".opr";
   }
 
-  std::ofstream file(filename_with_ext, std::ios::binary);
-  if (file.fail()) {
-    ERROR("Saving preset in \"%s\": %s", filename_with_ext, "file.fail()");
-    return ErrorMessage(
-        absl::StrFormat("Error opening the file \"%s\" for writing", filename_with_ext));
+  auto open_result = orbit_base::OpenFileForWriting(filename_with_ext);
+  if (open_result.has_error()) {
+    ERROR("Failed to open \"%s\": %s", filename_with_ext, open_result.error().message());
+    return ErrorMessage(absl::StrFormat("Failed to open \"%s\": %s", filename_with_ext,
+                                        open_result.error().message()));
```

Comment:
Maybe define a message and use it in ERROR and ErrorMessage (as it's done in line 716)? There also similar cases below

Reply:
done.